src/profile-logic/active-tab.js

```diff
@@ -0,0 +1,140 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
+import { getThreadSelectors } from '../selectors/per-thread';
+import { assertExhaustiveCheck } from '../utils/flow';
+
+import type { State } from '../types/state';
+import type { ThreadIndex, Pid } from '../types/profile';
+import type {
+  GlobalTrack,
+  LocalTrack,
+  TrackIndex,
+} from '../types/profile-derived';
+
+/**
+ * Take the global tracks and decide which one to hide during the active tab view.
+ * Some global tracks are allowed, some tracks are not, and we have to do some
+ * computations for some('process' type specifically).
+ */
+export function computeActiveTabHiddenGlobalTracks(
```

Comment:
What happens if this function hides all the global tracks?

Reply:
Yeah, bad UI interaction is better than a broken tool. This is an edge case at least anyway, so it's probably not as common.